common/logging/include/log-defines.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ * Copyright 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LOG_DEFINES
+#define LOG_DEFINES
+
+#ifndef TAG
+#define TAG ""
+#endif
+
+#define LOC_FMT " (%s:%d) "
+
+#define NRM "\x1B[0m"
+#define CYN "\x1B[36m"
+#define YEL "\x1B[33m"
+#define RED "\x1B[31m"
+
+#ifndef DO_DEBUG
```

Comment:
As you mentioned in discussion in PR #444, this is set only if SGX_BUILD is DEBUG, so it addresses the related aspect of issue #445.  Two things, though:
- might be good to add a comment here (or in the trusted logging.h) mentioning that in (PRE)RELEASE mode debug messages are guaranteed not to leak from enclave but any other log-message should not reveal anything sensitive?
- we also have a DEBUG flag from PDO (PDO_DEBUG_BUILD), we probably should add the setting (or in particular, unsetting) in `cmake/ConfigSGX.cmake`?

Reply:
> might be good to add a comment here (or in the trusted logging.h) mentioning that in (PRE)RELEASE mode debug messages are guaranteed not to leak from enclave but any other log-message should not reveal anything sensitive?

I have added some comments explaining that.
However, as I mentioned before, notice that `logging` (except for the `test`) does not use the `SGX_COMMON_CFLAGS`. The define is only necessary at compile-time when the ``log-defines.h" is included.
So, trusted logging, by itself, does not guarantee that there are no leaks.

Reply:
> we also have a DEBUG flag from PDO (PDO_DEBUG_BUILD), we probably should add the setting (or in particular, unsetting) in cmake/ConfigSGX.cmake?

I would be hesitant about that, because it would expose PDO details (that I think are not relevant) to FPC.
Right now, PDO_DEBUG_BUILD is only exposed in `common/crypto/CMakeLists.txt`, and disabled by default.
I needed that for the PDO integration, but hopefully it will not be used.

Reply:
ok